src/App.tsx

```diff
@@ -589,64 +575,53 @@ function App() {
       {messagesDisplays.map((messageDisplay) => {
         return (
           <div
+            className="text-align-center"
             key={getRetryableIdOrDepositHash(messageDisplay)}
             ref={resultRef}
           >
             {
               <>
                 <h3>
-                  Your transaction status on {messageDisplay.l2Network.name}
+                  L2{" "}
+                  {messageDisplay.l2TxHash !== "null" ? (
+                    <ExternalLink
+                      href={`${messageDisplay.explorerUrl}/tx/${messageDisplay.l2TxHash}`}
+                    >
+                      Transaction
+                    </ExternalLink>
+                  ) : (
+                    "Transaction"
+                  )}{" "}
+                  status on {messageDisplay.l2Network.name}
                 </h3>
+                {messageDisplay.l1ToL2Message !== undefined && (
+                  <ExternalLink
+                    href={`${
+                      messageDisplay.explorerUrl
+                    }/tx/${getRetryableIdOrDepositHash(messageDisplay)}`}
+                    showIcon
+                  >
+                    Check Your Retryable Ticket on{" "}
+                    {messageDisplay.l2Network.name}
+                  </ExternalLink>
+                )}
                 <p>{messageDisplay.text}</p>
-                {messageDisplay.showRedeemButton ? (
+                {messageDisplay.showRedeemButton && (
```

Comment:
just confirming that this won't render a stringified "null" or "undefined"? (I've seen that happen before but maybe it's on old JSX issue or something)

Reply:
no, try with this one:
https://retryable-tx-panel-git-styling-offchain-labs.vercel.app/tx/0x3b82e7582ec91694997f2dee2c030bbc0c149d98acd3503f4b55a50de20c3cf3